src/site/includes/header.html

```diff
@@ -67,8 +67,6 @@
     <script nonce="**CSP_NONCE**" src="{{ botframework_cdn }}"></script>
   {% endif %}
 
-  {% include "src/site/includes/coronavirus-schema-org.json.liquid" %}
```

Comment:
Removing this, because it created schema.org data that wasn't reflected visually on the page, so it was considered invalid.